tokens/colors.js

```diff
@@ -6,94 +6,139 @@
 //
 // *----------------------------------*/
 
-// Navy and white
-const colorNavy = 'hsl(205deg 79% 11%)';
-const colorWhite = 'hsl(0deg 0% 100%)';
-
-// Gray
-const colorGray = 'hsl(0deg 0% 87%)';
-
-// Bright colours
+// Brand colours
 const colorPink = 'hsl(335deg 85% 49%)';
+const colorNavy = 'hsl(205deg 79% 11%)';
+const colorTeal = 'hsl(175deg 100% 33%)';
+const colorLightOrange = 'hsl(31deg 94% 65%)';
 const colorCandyPink = 'hsl(334deg 79% 89%)';
+const colorCyan = 'hsl(179deg 92% 77%)';
+const colorBlue = 'hsl(237deg 41% 44%)';
 const colorOrange = 'hsl(11deg 81% 55%)';
-const colorLightOrange = 'hsl(31deg 94% 65%)';
 const colorLemon = 'hsl(56deg 100% 73%)';
-const colorBlue = 'hsl(237deg 41% 44%)';
-const colorTeal = 'hsl(175deg 100% 36%)';
-const colorCyan = 'hsl(179deg 92% 77%)';
+
+// Neutral colours
+const colorWhite = 'hsl(0deg 0% 100%)';
+const colorLightestGrey = 'hsl(0deg 0% 98%)';
+const colorSoftGrey = 'hsl(0deg 0% 96%)';
+const colorPaleGrey = 'hsl(0deg 0% 93%)';
+const colorGrey = 'hsl(0deg 0% 87%)';
+const colorMidGrey = 'hsl(0deg 0% 79%)';
+const colorDarkGrey = 'hsl(0deg 0% 73%)';
+const colorDarkestGrey = 'hsl(0deg 0% 60%)';
+
+// Semantic colours
+const colorError = 'hsl(0deg 93% 48%)';
+const colorError20 = 'hsl(0deg 93% 55% / 20%)';
```

Comment:
will need to change this also to include the same red variation as `colorError` whilst maintaining the 20%
```suggestion
const colorError20 = 'hsl(0deg 93% 48% / 20%)';
```